wled00/xml.cpp

```diff
@@ -10,7 +10,7 @@
 //build XML response to HTTP /win API request
 void XML_response(AsyncWebServerRequest *request, char* dest)
 {
-  char sbuf[(dest == nullptr)?1024:1]; //allocate local buffer if none passed
+  char sbuf[(dest == nullptr)?1024:1]; // allocate local buffer if none passed ... (GCC extension to have variable-sized stack array)
```

Comment:
this is not new code  ... I just added a comment that this is using a GCC-specific extension, since standard C++ does not permit a stack-allocated array to have variable element count.